pkg/core/hash.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 Pusher Ltd.
+Copyright 2018, 2019 Pusher Ltd.
```

Comment:
Same as above, this doesn't need changing

Reply:
Will revert